lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
```

Comment:
For documentation purposes, I made an attempt at shelling out and running with `BUNDLE_WITHOUT`, but I got a whole load of other Bundler issues from that.

For now, this feature works for Bundler 1.9.4 and higher, and falls back to just using `bundle install` for older versions.